src/pages/stage-page/StagePage.tsx

```diff
@@ -40,7 +40,7 @@ export const StagePage = ({
         className="govuk-heading-xl govuk-!-margin-top-6"
         data-testid="form-title"
       >
-        {title} Stage
```

Comment:
I have removed the word "stage" from here as not all stages display it after the name of the stage on this page. I have added "stage" to the titles of the stages where applies (see stages.tsx)